src/Controls/tests/DeviceTests/Elements/NavigationPage/NavigationPageTests.cs

```diff
@@ -303,9 +303,33 @@ TabbedPage CreateTabbedPage(string title) => new TabbedPage()
 			});
 		}
 
-		[Fact(DisplayName = "NavigationPage Does Not Leak")]
+		[Fact(DisplayName = "Does Not Leak")]
```

Comment:
Pass on iOS but is failing on Windows:
`Reference to Microsoft.Maui.Controls.NavigationPage (type Microsoft.Maui.Controls.NavigationPage is still alive.`
We could create a specific issue to review the leak on Windows and ignore the test on this platform for now.

Reply:
Let me just spend a minute to see if I can fix Windows, too.